src/mscorlib/src/System/Attribute.cs

```diff
@@ -431,7 +431,7 @@ private static AttributeUsageAttribute InternalGetAttributeUsage(Type type)
                 return AttributeUsageAttribute.Default;
 
             throw new FormatException(
-                Environment.GetResourceString("Format_AttributeUsage", type));
+SR.Format(SR.Format_AttributeUsage, type));
```

Comment:
Yet again, trivia is not trivial in roslyn syntax trees. I replaced the call expression without fixing up the trivia. Ill fix all these